tests/services/getQuests.test.js

```diff
@@ -0,0 +1,77 @@
+import { getQuests } from "@services/apiService";
+
+const API_URL = process.env.NEXT_PUBLIC_API_LINK;
+
+global.fetch = jest.fn();
+
+describe("getQuests", () => {
+  beforeEach(() => {
+    fetch.mockClear();
+  });
+
+  it("should fetch and return quests data", async () => {
+    const mockData = [
```

Comment:
i dont think this a valid returned data. Can you please test the endpoint for this and add that data here for verification. You will also need to add it as a return type in the api call in apiServices.ts